src/FieldCanvas.h

```diff
@@ -1,5 +1,6 @@
+#pragma once
```

Comment:
Is this a stylistic change that you'd recommend going forward? Placing the `#pragma once` above the file comments?

Reply:
I've been adopting that.  It's similar to the #!/bin/sh at the top of scripts...

Reply:
Okay, I'm adding that to https://github.com/calband/calchart/issues/282 as a change we should consider auditing in other files as well.

I haven't seen coding style choice before -- do you have any other examples of this in the wild? It looks similar to a shebang line, but has a very different function. My main concern is that it will be easily overlooked up here; but then again, I've never worked with a project that did this so I could be very wrong about that.

Reply:
#import starts making this less important...  I've started seeing it in other types of projects.  I think modules and better tools are going to make this obsolete.